core/state_transition.go

```diff
@@ -211,15 +216,20 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 	} else {
 		// Increment the nonce for the next transaction
 		st.state.SetNonce(msg.From(), st.state.GetNonce(sender.Address())+1)
-		ret, st.gas, vmerr = evm.Call(sender, st.to(), st.data, st.gas, st.value)
+		ret, st.gas, vmerr = evm.Call(sender, st.to(), st.data, st.gas, st.value, st.txType)
```

Comment:
Again, can we not report the insufficient fund error, by changing the code in evm.Call?

Reply:
now we are removing txType for addition only